Google.PowerShell.IntegrationTests/SQL/SQL.GcSqlInstances.Tests.ps1

```diff
@@ -73,6 +73,24 @@ Describe "Add-GcSqlInstance" {
         $myInstance.Settings.MaintenanceWindow.Hour | Should Be 2
         gcloud sql instances delete $instance --quiet 2>$null
     }
+
+    It "should be able to create a read replica instance" {
+        $r = Get-Random
+        # A random number is used to avoid collisions with the speed of creating
+        # and deleting instances.
+        $instance = "test-inst$r"
+        $setting = New-GcSqlSettingConfig "db-n1-standard-1" 
+        $replicaConfig = New-GcSqlInstanceReplicaConfig
+        $config = New-GcSqlInstanceConfig $instance -SettingConfig $setting `
+            -ReplicaConfig $replicaConfig -MasterInstanceName "test-db2"
+        Add-GcSqlInstance $config
+        $newInstances = Get-GcSqlInstance
```

Comment:
Maybe add a line of vertical whitespace here for readability? Currently this is a big wall of text.